pkg/controller/testing/builder/configuration.go

```diff
@@ -0,0 +1,62 @@
+package builder
```

Comment:
Haven't thought about it too carefully, but would it make sense for these methods to be defined in the api package (or a subpackage of that) rather than in the controller testing package?

Reply:
That was @mattmoor's suggestion here:
https://github.com/knative/serving/pull/1762#discussion_r206896610